hash/gjrstar.xml

```diff
@@ -3,20 +3,60 @@
 <!--
 license:CC0-1.0
 -->
-<softwarelist name="gjrstar" description="VTech Genius Junior Redstar cartridges">
+<softwarelist name="gjrstar" description="VTech Genius Junior Redstar (and clones) cartridges">
 
 	<!-- According to the boxart, this is compatible with: Junior Redstar/Redstar 2/Redstar 3 -->
 	<software name="wortspie">
 		<description>Wortspiele</description>
 		<year>200?</year>
 		<publisher>Video Technology</publisher>
 		<info name="serial" value="80-10171" />
-
 		<part name="rom" interface="genius_rom">
 			<dataarea name="rom" size="0x40000">
 				<rom name="27-5883-00" size="0x40000" crc="99214bce" sha1="75294fbc411b0b39b3db4727824faf29127675af"/>
 			</dataarea>
 		</part>
 	</software>
 
+	<!-- Russian carts for Precomputer Electronik (Прекомпьютер Электроник), compatibility with clones from other regions is unknown -->
+
+	<software name="matematika">
+		<description>Matematika - Podarok Pervoklassniku</description> <!-- Mathematics - Gift for a First Grader -->
+		<year>????</year>
+		<publisher>Video Technology / DjiNo</publisher> <!-- VTech / ДжиНо -->
+		<info name="serial" value="80-10126-04" /> <!-- Nº 4 -->
+		<info name="alt_title" value="Математика - Подарок Первокласснику"/>
+		<part name="rom" interface="genius_rom">
+			<dataarea name="rom" size="0x10000">
+				<rom name="80-10126-04.bin" size="0x10000" crc="5c0c2e30" sha1="b75119188354833695c1e9a7686ac244b474ff09"/> <!-- 27C512 inside the cart, second half empty -->
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="nazagadki">
+		<description>Narodnyye Zagadki</description> <!-- Folk Riddles -->
+		<year>????</year>
+		<publisher>Video Technology / DjiNo</publisher> <!-- VTech / ДжиНо -->
+		<info name="serial" value="80-10126-08" /> <!-- Nº 8 -->
+		<info name="alt_title" value="Народные Загадки"/>
+		<part name="rom" interface="genius_rom">
+			<dataarea name="rom" size="0x10000">
+				<rom name="80-10126-08.bin" size="0x10000" crc="1c667c44" sha1="32f7a33a6ae95dc9b68e215e73a8c4a203d39033"/> <!-- 27C512 inside the cart, second half empty -->
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="poiskakhds">
+		<description>B Poiskakh Dukhovnykh Sokrovishch</description> <!-- In Search of Spiritual Treasures -->
```

Comment:
The first word should be transliterated as “V”.

Reply:
Fixed!